src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -1,9 +1,13 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import Scroll from 'react-scroll';
-import { connect } from 'react-redux';
 import { withRouter } from 'react-router';
 
+import DowntimeNotification, {
+  externalServiceStatus,
+} from 'platform/monitoring/DowntimeNotification';
+import DowntimeMessage from 'platform/monitoring/DowntimeNotification/components/Down';
```

Comment:
This is importing the `<Down>` component instead of `src/platform/forms/save-in-progress/DowntimeMessage`. It is the only real difference between this and what `RoutedSavableReviewPage` is doing.

https://github.com/department-of-veterans-affairs/vets-website/blob/d412bc0a1834ff507761f4be7f4d94eebb876349/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L22